database/factories/ModelFactory.php

```diff
@@ -76,6 +78,13 @@
     ];
 });
 
+$factory->define(Setting::class, function ($faker) {
```

Comment:
> Seem so your tests rely on it, it makes sens to have it.

I don't really need everything `cachet:seed` creates for my tests. I usually create my own test data. Due to the way Cachet works when it's first being set up, I need to bypass that step by those two settings using the factories.

> I always expect factory built models to be randomly hydrated

If you want, I can supply those values as a parameter to the create method of the Setting factory.